linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -466,6 +466,22 @@ pipeline {
             always {
               junit "Report\\*-junit.xml"
               archiveArtifacts "TestResults\\**\\*"
+              script {
+                 withCredentials(bindings: [
+                 file(credentialsId: 'CBS_Azure_Secrets_File',
+                       variable: 'Azure_Secrets_File')
+                ]) {
+                        def failures=powershell returnStdout: true, script: '''$content = [xml](Get-Content ".\\Report\\*-junit.xml")
```

Comment:
please check reportStageStatus method and either remove that function, but keep the unicity of the pipeline run as it was defined there, either remove the database upload code that you have here and reuse that method. We shouldn t have code that does the same stuff in a different way in the codebase.

Reply:
@ader1990 reportStageStatus works against the Linux slave, I use this one since it run against windows slave